accessclinicaldata.niaid.nih.gov/manifests/fence/fence-config-public.yaml

```diff
@@ -182,4 +182,19 @@ STORAGE_CREDENTIALS: []
 #       length restrictions on group names.
 GOOGLE_GROUP_PREFIX: 'gen3'
 
+S3_BUCKETS:
```

Comment:
fix `ACCESS_TOKEN_EXPIRES_IN`, `REFRESH_TOKEN_EXPIRES_IN` and `S3_BUCKETS` in the QA config file too

Reply:
👍

Reply:
Here is the PR to update the QA config file
https://github.com/uc-cdis/gitops-qa/pull/1442